src/board.cpp

```diff
@@ -102,6 +102,11 @@
 #define BOARD_ID        {1, 1, 1, 1}
 #define BOARD_SECRET    {1, 1, 1, 1, 1, 1, 1, 1, 1}
 
+#else // default for all other boards
```

Comment:
This allow to build without modifying the lib, most make won't bother with this, I guess.

Reply:
This is certainly worth a chance, and I think the same ID should be used for the `-DUSE_TINYUSB` series.